esmvaltool_sample_data/datasets.yml

```diff
@@ -27,3 +27,7 @@ ignore:
   - CMIP6.CMIP.EC-Earth-Consortium.EC-Earth3-Veg-LR.historical.r1i1p1f1.Amon.ta.gr.v20200217
   # something wrong with lon coord
   - CMIP6.CMIP.UA.MCM-UA-1-0.historical.r1i1p1f1.Amon.ta.gn.v20190731
+  # iris.exceptions.ConcatenateError: failed to concatenate into a single cube.
```

Comment:
Nice that so much of the data at least makes it through loading and the regridder now!

Reply:
Yeah, the suggestion to use iris for regridding and making files with at least 2x2 helped a lot!